src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5362,7 +5362,8 @@ void parseStatLine(const std::string &lineIn,
                    bool &desConditionlinepassed,
                    bool &heatingDesignlinepassed,
                    bool &coolingDesignlinepassed,
-                   bool &isKoppen)
+                   bool &isKoppen,
+                   bool &insideLiquidPrecipitation)
```

Comment:
Passing another boolean here makes me squirm a little, but I recognize that this is _not_ something that is going to be changing almost ever, so this is ... OK :smile: